### DISCUSSION TITLE
An issue I created appears deleted #570

### DISCUSSION THREAD

**Discussion by Pilgrim1379** [marlonrichert/zsh-autocomplete#572] [General]
I've been unable to use zsh-autocomplete since [commit](784e3a9f8b4dd01c49ed93356b5a1f5cc199ce54). I created an issue with the number 570. It now appears to have been deleted for some reason since I can't find it in the open or closed issues.

If is was deleted, it'll be nice to have an explanation so I don't repeat any errors I might have made.

Thanks

**Comment by ghost** [comment]
Found it in your overview, and the issue was transferred to https://github.com/marlonrichert/zsh-snap/issues/231

**Reply by Pilgrim1379** [reply]
Thank you.

**Comment by Pilgrim1379** [comment]
Closed
